docs/ModelDescriptors.toml

```
@@ -233,19 +238,21 @@ SVMNuRegressor_MLJScikitLearnInterface = ["regression"]
 SVMRegressor_MLJScikitLearnInterface = ["regression"]
 SimpleImputer_BetaML = ["missing_value_imputation"]
 SpectralClustering_MLJScikitLearnInterface = ["clustering", "static_models"]
-Standardizer_MLJModels = ["encoders"]
+Standardizer_MLJTransforms = ["encoders"]
 SubspaceLDA_MultivariateStats = ["classification", "dimension_reduction"]
+SumEncoder_MLJTransforms = ["encoders"]
```

This is a special case of `ContrastEncoder` so if we are to include it here then we can invlude the remaining special cases as well:
```suggestion
SumEncoder_MLJTransforms = ["encoders"]
DummyEncoder_MLJTransforms = ["encoders"]
HelmertEncoder_MLJTransforms = ["encoders"]
ForwardDifferenceEncoder_MLJTransforms = ["encoders"]
HypothesisEncoder_MLJTransforms = ["encoders"]
```